mlir/test/Dialect/Tosa/availability.mlir

```diff
@@ -622,6 +622,28 @@ func.func @test_rescale(%arg0: tensor<13x21x3x!quant.uniform<u8:f32, 0.015655439
   return %0 : tensor<13x21x3x!quant.uniform<i8:f32, 0.015655439347028732:-1>>
```

Comment:
A little unrelated to this PR - I feel this test case should be considered invalid since `input_unsigned=false`. This is something we should check in the verifier.